SpatialGDK/Source/My_APL.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What does this file do and is it named correctly?

Reply:
Unreal uses this file to copy over binaries on the phone. Will rename the file